docs/src/docs/deploy.md

```diff
@@ -0,0 +1,112 @@
+# Deploying ModelKits
+
+This page outlines how to use `init` or Kit CLI containers to deploy a ModelKit-packaged AI/ML project to Kubernetes or any other container runtime. The KitOps repo provides pre-built ModelKits that can be used for both semi-turnkey solutions, and more DIY options.
+
+## Pre-built Containers
+
+There are currently two pre-built containers:
+
+1. Init container: https://github.com/jozu-ai/kitops/blob/main/build/dockerfiles/init/README.md
+1. Kit CLI container: https://github.com/jozu-ai/kitops/blob/main/build/dockerfiles/README.md 
+
+## Init Container
+
+The init container unpacks the model reference from a ModelKit to a specific path and then exits. This makes it useful as a Kubernetes `init` container. This container also supports verifying signatures for containers automatically from key-based or keyless signers.
+
+The container is configurable via environment variables:
+
+`$MODELKIT_REF`: The ModelKit to pull (required).
+`$UNPACK_PATH`: Where to unpack the ModelKit (normally you’d want a `volumeMount` here). This is required and will default to `/home/user/modelkit`.
+`$UNPACK_FILTER`: Optional filter to limit what is unpacked (e.g., just the model, or model + code). The filter format is the same as the [–filter command line argument](./cli/cli-reference.md) for the Kit CLI.
```

Comment:
I think your editor replaced `--` with an em-dash here, which would break copy pasting (if someone happened to do that)
```suggestion
`$UNPACK_FILTER`: Optional filter to limit what is unpacked (e.g., just the model, or model + code). The filter format is the same as the [`--filter` command line argument](./cli/cli-reference.md) for the Kit CLI.
```